app/views/idv/fail.html.slim

```diff
@@ -5,4 +5,4 @@ h1.h3.my0 = t('idv.titles.hardfail')
 p.mb1 = t('idv.messages.hardfail')
 p.mb1 = t('idv.messages.hardfail2', hours: Figaro.env.idv_attempt_window_in_hours)
 p.mb1 = t('idv.messages.hardfail3')
-p.mb1 = t('idv.messages.hardfail4')
+p.mb1 = t('idv.messages.hardfail4', sp: @sp_name)
```

Comment:
Two versions is what we'll need for launch. Where should @jeanninehunter put the "no SP" version?